### PR TITLE
Remove spurious log message from istioctl output

### PR DESCRIPTION
**Pull request by myshkin5**
**Please provide a description of this PR:**

### CODE REVIEW

File: pkg/kube/portforwarder.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ func (f *forwarder) Start() error {
 				f.errCh <- fmt.Errorf("port forward: %v", err)
 				return
 			}
-			scopes.Framework.Infof("port forward completed without error")
+			scopes.Framework.Debugf("port forward completed without error")
```
[the remark this answers is not in the document]
Comment (hanxiaop) [reply pkg/kube/portforwarder.go:88]: Having the test framework log in normal code seems to be a bug.

[the remark this answers is not in the document]
Comment (myshkin5) [reply pkg/kube/portforwarder.go]: @hanxiaop Got similar results switching to `istio.io/pkg/log` but I just changed this one line.

Let me know if you would like me to update all references to `scopes.Framework` in this file. Thanks!

[the remark this answers is not in the document]
Comment (hanxiaop) [reply pkg/kube/portforwarder.go]: Yes could you please update all the references? I think after updating to the `istio.io/pkg/log` package, the PR is good to go. Thanks!

[the remark this answers is not in the document]
Comment (myshkin5) [reply pkg/kube/portforwarder.go]: Looks like there was just one more reference to the test logging package. I switched it to standard logging in the latest commit.

**Review by hanxiaop — APPROVED** [review]

File: pkg/kube/portforwarder.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (hanxiaop) [reply pkg/kube/portforwarder.go]: Looks good, thanks!

**Pull request merged**